yourgov/components/FormMobileFoodVendorPermit/UploadFileTable.tsx

```diff
@@ -0,0 +1,217 @@
+									>
+										{label}
+									</TableHeader>
+								);
+							})}
+						</TableRow>
+					</TableHead>
+					<TableBody>
+						{loading ? (
+							<Fragment>
+								{Array.from(Array(10).keys()).map((i) => (
+									<TableRow key={i}>
+										<TableCell>
+											<SkeletonText />
+											<VisuallyHidden>Loading</VisuallyHidden>
+										</TableCell>
+										<TableCell>
+											<SkeletonText />
+											<VisuallyHidden>Loading</VisuallyHidden>
+										</TableCell>
+										<TableCell>
+											<SkeletonText />
+											<VisuallyHidden>Loading</VisuallyHidden>
+										</TableCell>
+										<TableCell>
+											<SkeletonText />
+											<VisuallyHidden>Loading</VisuallyHidden>
+										</TableCell>
+										<TableCell>
+											<SkeletonBox height={32} />
+											<VisuallyHidden>Loading</VisuallyHidden>
+										</TableCell>
+									</TableRow>
+								))}
+							</Fragment>
+						) : (
+							<Fragment>
+								{documents.map((document, index) => {
+									const { id, documentType, error, file, size } = document;
+									// Adding 2 because the table header row is the first row
+									const rowIndex = index + 2;
+									return (
+										<TableRow
+											aria-rowindex={rowIndex}
+											invalid={error}
+											key={documentType}
+										>
+											<TableCell as="th" scope="row">
+												{documentType}
+											</TableCell>
+											<TableCell>
+												<Box>{file}</Box>
+											</TableCell>
+											<TableCell>
+												{error ? (
+													<StatusBadge
+														label="File missing"
+														appearance="subtle"
+														tone="errorHigh"
+													/>
+												) : (
+													size
+												)}
+											</TableCell>
+											<TableCell>
+												<Button
+													aria-label={`${
+														size ? 'Remove' : 'Upload'
+													}: ${documentType}`}
+													focusRingFor="all"
+													iconBefore={size ? DeleteIcon : UploadIcon}
+													id={id}
+													onClick={() =>
+														size
+															? removeFile(document)
+															: openDrawer({
+																	id,
+																	documentType,
+																	error,
+																	file,
+																	size,
+															  })
+													}
+													variant="text"
+												>
+													{size ? 'Remove' : 'Upload'}
+												</Button>
+											</TableCell>
+										</TableRow>
+									);
+								})}
+							</Fragment>
+						)}
+					</TableBody>
+				</Table>
+			</TableWrapper>
+		</Stack>
+	);
+});
+
+const headers: {
+	label: string;
+	width?: TableHeaderProps['width'];
+	textAlign?: TableHeaderProps['textAlign'];
+}[] = [
+	{
+		label: 'Document type',
+		width: { xs: '20rem', lg: 'auto' },
+	},
+	{
+		label: 'File',
+		width: '12rem',
+	},
+	{
+		label: 'Size',
+		width: { xs: '16rem', lg: 'auto' },
+	},
+	{
+		label: 'Label',
```

Comment:
This should be "Action" or "Upload" or something? The design seems wrong

Reply:
Went for `Action` because the label changes to "Remove"